tests/tests.rs

```diff
@@ -211,3 +370,35 @@ fn test_provider_registry() {
         ]
     )
 }
```

Comment:
@rvanasa : additional test ideas (also ok if you prefer to do that in another MR): 

1. ensure that the json is correctly sanitized. E.g, query two different providers with the same JSON-RPC request such that both providers return the same JSON content but the string value is different (e.g. fields are ordered differently). Ensure that results provided by the rpc canister are equal in terms of string equality.
2. Ensure that unknown fields in the response are ignored. This ensures that things will keep working if there is a new version of the Ethereum JSON-RPC API

Reply:
:heavy_plus_sign:

Reply:
Sounds good! I added (1.) in this pull request and will include (2.) separately with other Candid-RPC tests.